### PR TITLE
Create eip-1261

### PR DESCRIPTION
**Pull request by chaitanyapotti**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-X.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your Github username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-1261.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
+---
+eip: 1261
+title: ERC-1261
```
Comment (Arachnid) [inline EIPS/eip-1261.md]: This needs a more descriptive name.

File: EIPS/eip-1261.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
+---
+eip: 1261
+title: ERC-1261 
+author: Chaitanya Potti <chaitanya@electus.network>, Partha Bhattacharya <partha@electus.network>
+type: Standards Track
+category: ERC
+status: Draft
+created: 2018-07-14
+requires: 
+---
```
Comment (Arachnid) [inline EIPS/eip-1261.md]: Please supply a discussions-to URL.

File: EIPS/eip-1261.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+eip: 1261
```
Comment (Arachnid) [inline EIPS/eip-1261.md]: Why 1261?

Comment (chaitanyapotti) [reply EIPS/eip-1261.md]: No reason.  It's the issue number. What do you recommend?

File: EIPS/eip-1261.md (hunk 1 above)
Comment (chaitanyapotti) [reply EIPS/eip-1261.md]: Done

File: EIPS/eip-1261.md (hunk 2 above)
Comment (chaitanyapotti) [reply EIPS/eip-1261.md]: Done

**Pull request merged**